corehq/form_processor/signals.py

```diff
@@ -0,0 +1,4 @@
+from django.dispatch.dispatcher import Signal
+
+
+sql_case_post_save = Signal(providing_args=["case"])
```

Comment:
Is it necessary to have a new signal? We could just use the same one with a different sender.